TACC/protx-dashboard
# Task/COOKS-326: enable front end unit tests

## Overview: ##

Enable unit tests locally and in CI

## Related Jira tickets: ##

* [COOKS-326](https://jira.tacc.utexas.edu/browse/COOKS-326)

## Summary of Changes: ##

## Testing Steps: ##
1. Run `npm install` locally
2. Run `npm run test` locally
3. Confirm that CI is running
